src/index.js

```diff
@@ -15,8 +17,7 @@ function DangerouslySetHtmlContent({ html, dangerouslySetInnerHTML, ...rest }) {
     divRef.current.appendChild(slotHtml) // Append the new content
   }, [html, divRef])
 
-  // eslint-disable-next-line react/react-in-jsx-scope
-  return <div {...rest} ref={divRef} />
+  return createElement('div', { ...rest, ref: divRef })
```

Comment:
This is not transpiled anymore but it's not a problem, it's supported everywhere now:
- https://caniuse.com/mdn-javascript_operators_spread
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax#browser_compatibility

Reply:
The arrow function above is also not transpiled anymore but this is supported everywhere:
- https://caniuse.com/arrow-functions
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#browser_compatibility

Same for `const`:
- https://caniuse.com/const
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const#browser_compatibility